busybox/busybox.py

```diff
@@ -9,39 +9,63 @@
 
 from trunner.ctx import TestContext
 from trunner.dut import Dut
-from trunner.types import TestResult, TestSubResult, Status
+from trunner.types import TestResult, Status
+
+# Example of parsed outputs:
```

Comment:
use multiline string , right now I don't know if the input has spaces in front or not, eg:
```python
EXAMPLE_INPUT = """
test-msg-line-1
test-msg-line-2
FAIL: test-name-1
fail-msg-line-1
fail-msg-line-2
---Testcase test-name-2 starting---
test-msg-line-1
test-msg-line-2
PASS: test-name-2
---Testcase test-name-3 starting---
SKIPPED: test-name-3
"""
```
this would also make it easier to write unit tests (use  variable as an input).

Also - test names should be in `'`?